### ISSUE TITLE
Bug: securitySchemes oauth2 empty scopes is stripped

### ISSUE THREAD

**Issue by khart-twilio** [pb33f/libopenapi#289]
input:
```
components:
  securitySchemes:
    someName:
      type: oauth2
      flows:
        clientCredentials:
          tokenUrl: https://oauth.somewhere.com/v1/token
          scopes: {}
```
invalid output (`scopes` key is removed but is required as per https://swagger.io/docs/specification/authentication/oauth2/):
```
components:
  securitySchemes:
    someName:
      type: oauth2
      flows:
        clientCredentials:
          tokenUrl: https://oauth.somewhere.com/v1/token
```

can this be fixed, and in the meantime what is the best way to work around the issue?

Thanks for the library, I love the website style :)

**Comment by daveshanley** [comment]
Fixed in `v0.16.6` Thank you for your contribution!

**Comment by khart-twilio** [comment]
Thanks, I was digging into the tests but it takes a while to get familiar.  I appreciate you taking it up!
